rest-service/manager_rest/rest/endpoint_mapper.py

```diff
@@ -127,7 +127,8 @@ def setup_resources(api):
         'Sites': 'sites',
         'SitesName': 'sites/<string:name>',
         'ClusterStatus': 'cluster-status',
-        'DeploymentsLabels': 'labels/deployments'
+        'DeploymentsLabels': 'labels/deployments',
+        'DeploymentsLabelsKey': 'labels/deployments/<string:key>'
```

Comment:
Here, you dropped a ','

Reply:
Thanks, will add it. I need to get used to it